### PR TITLE
Added conversion of DER to PEM for verify

### LINKED ISSUE #157: add support for binary-encoded x509 certs
`notation verify` only supports validating with ASCII-encoded x509 certs (PEM)

Today, you can convert a binary encoded DER to PEM and verify using PEMs:
`openssl x509 -in mycert.crt -inform DER -out mycert.pem`

However, users shouldn't need to do this conversion manually and notation should add support for binary-encoded x509 certs

### PR DESCRIPTION
**Pull request by patrickzheng200**
Signed-off-by: Patrick Zheng <patrickzheng@microsoft.com>

Resolves #157

### CODE REVIEW

File: pkg/signature/jws.go [hunk 1]
```diff
@@ -52,7 +53,10 @@ func NewVerifierFromFiles(certPaths []string) (*jws.Verifier, error) {
 			return nil, err
 		}
 		if !verifier.VerifyOptions.Roots.AppendCertsFromPEM(data) {
-			return nil, fmt.Errorf("failed to parse PEM certificate: %q", path)
+			certPEM := pem.EncodeToMemory(&pem.Block{Type: "CERTIFICATE", Bytes: data})
```
Comment (binbin-li) [inline pkg/signature/jws.go:56]: Probably we can add a comment to explain that we're trying to convert DER to PEM? But it looks good to me overall.

Comment (patrickzheng200) [reply pkg/signature/jws.go:56]: > Probably we can add a comment to explain that we're trying to convert DER to PEM? But it looks good to me overall.

Sure. Will add a comment to it.

Comment (JeyJeyGao) [reply pkg/signature/jws.go:56]: Did you test it with some corner cases? like empty file, wrong cert file.

Comment (patrickzheng200) [reply pkg/signature/jws.go:57]: > Did you test it with some corner cases? like empty file, wrong cert file.

Yes, both scenarios are covered. Also tested end to end with the example net-monitor image.

**Comment by SteveLasker** [comment]
@patrickzheng200, was this accidently closed, vs. merged?

**Comment by patrickzheng200** [comment]
> @patrickzheng200, was this accidentally closed, vs. merged?

I noticed that pkg/signature/jws.go file has been removed from notation. Do we still need this PR?

**Comment by patrickzheng200** [comment]
We are supporting notation verify with binary encoded DER in [notation-core-go](https://github.com/notaryproject/notation-core-go/blob/1d963fd57141fedb192a064867fb3114e43220cb/x509/cert.go#L24).
